### PR TITLE
fix(tvos): fix tv os build and sample

### LINKED ISSUE #3779: [BUG]:  updatesNowPlayingInfoCenter' is unavailable in tvOS when using react-native-video in react-native-tvos App
### Version

6.0.0

### What platforms are you having the problem on?

Apple tvOS

### Architecture

New architecture with interop layer

### What happened?

When I am trying to run the project on tvOS, I am getting a build failed error. The error 'updatesNowPlayingInfoCenter' is unavailable in tvOS is occurring because the react-native-video library is attempting to set the updatesNowPlayingInfoCenter property on the RCTVideoPlayerViewController class, which is not available or supported on the tvOS platform. It was working well when using react native video 5.2.1

MY Package JSON
```

{
  "name": "searchbox",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "android": "expo run:android",
    "ios": "expo run:ios",
    "web": "expo start --web",
    "start": "expo start",
    "tvos": "expo run:ios --scheme SearchBox-tvOS --device \"Apple TV\"",
    "lint": "eslint .",
    "test": "jest"
  },
  "dependencies": {
    "@react-navigation/native": "^6.1.17",
    "expo": "^50.0.2",
    "react": "18.2.0",
    "react-native": "npm:react-native-tvos@^0.73.2-0",
    "react-native-safe-area-context": "^4.10.1",
    "react-native-screens": "^3.31.1",
    "react-native-video": "^6.0.0"
  },
  "devDependencies": {
    "@babel/core": "^7.20.0",
    "@babel/preset-env": "^7.20.0",
    "@babel/runtime": "^7.20.0",
    "@react-native/babel-preset": "^0.73.19",
    "@react-native/eslint-config": "^0.73.2",
    "@react-native/metro-config": "^0.73.3",
    "@react-native/typescript-config": "^0.73.1",
    "@types/react": "^18.2.6",
    "@types/react-test-renderer": "^18.0.0",
    "babel-jest": "^29.6.3",
    "eslint": "^8.19.0",
    "jest": "^29.6.3",
    "prettier": "2.8.8",
    "react-test-renderer": "18.2.0",
    "typescript": "5.0.4"
  },
  "engines": {
    "node": ">=18"
  }
}

```
Xcode Version 15.3

### Reproduction

repository link

### Reproduction

1. Initialize a new React Native project for tvOS with TypeScript using the official tvOS template:
```
yarn create react-native-app TestApp --template https://github.com/react-native-tvos/react-native-template-typescript-tv/tree/tv-release-0.73.0 --template-path template
cd TestApp
```

2. Install `react-native-video` by running `yarn add react-native-video`.

3. Navigate to the `ios` folder and run `pod install` to install the required iOS dependencies.

4. In Xcode, open the project's tvOS target and go to the "General" tab. Under the "Linked Frameworks and Libraries" section, add `libreact-native-video.a`.

5. Build the app for tvOS using the Expo CLI command:
```
yarn tvos ("expo run:ios --scheme SearchBox-tvOS --device \"Apple TV\"")
```

After following these steps, you should encounter the same error: `'updatesNowPlayingInfoCenter' is unavailable in tvOS` when attempting to build the app for the tvOS platform using the Expo CLI command.

### PR DESCRIPTION
**Pull request by freeboub**
## Summary
Fix build issue on tvOS and ensure sample is working

### Motivation
Fix: https://github.com/TheWidlarzGroup/react-native-video/issues/3779

### Changes
Conditionnal build patch

## Test plan
build and run sample tvOS app

### CODE REVIEW

File: ios/Video/RCTVideo.swift [hunk 1]
```diff
@@ -1045,7 +1045,9 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     func createPlayerViewController(player: AVPlayer, withPlayerItem _: AVPlayerItem) -> RCTVideoPlayerViewController {
         let viewController = RCTVideoPlayerViewController()
         viewController.showsPlaybackControls = self._controls
-        viewController.updatesNowPlayingInfoCenter = false
+        #if !os(tvOS)
+            viewController.updatesNowPlayingInfoCenter = false
+        #endif
```
Comment (KrzysztofMoch) [inline ios/Video/RCTVideo.swift:1050]: I don't like it... docs say it should work... I will look into this - give me time

Comment (freeboub) [reply ios/Video/RCTVideo.swift:1050]: I also don't really like these #if ...
I agree doc say it is supported, but when I go to definition from xcode, It doesn't say it is support on tvOS ...

File: examples/exampletvOS/ios/Podfile [hunk 2]
```diff
@@ -9,7 +9,7 @@ production = ENV["PRODUCTION"] == "1"
 
 target 'exampletvOS' do
 
-  platform :ios, min_ios_version_supported
```
Comment (freeboub) [inline examples/exampletvOS/ios/Podfile:12]: @KrzysztofMoch I think we need to document this point also ! 
Would that fix be better ? https://github.com/TheWidlarzGroup/react-native-video/issues/3737#issuecomment-2096624471

Comment (KrzysztofMoch) [reply examples/exampletvOS/ios/Podfile:12]: Yes, I am also thinking about adding this info to README (that min ios version is 13) and to upgrade helper

File: ios/Video/RCTVideo.swift (hunk 1 above)
Comment (KrzysztofMoch) [reply ios/Video/RCTVideo.swift:1050]: So I have double checked that and it's it as you say - strange 
Let's merge this

**Pull request merged**